.travis.yml

```diff
@@ -43,6 +45,11 @@ before_install:
     fi
 
 install:
+  - pip install virtualenv
```

Comment:
All of these should not be here. Travis is already creating a venv for us. I think that you adding `language: 'node_js'` above may have changed this. I would strongly prefer it you remove the `language: 'node_js' from above and find another way to specify a node_js version in Travis that does not force you to add all the stuff about python you added.

Reply:
Okay, doing that now ...